bundles/AdminBundle/Resources/views/Admin/DataObject/DataObject/previewVersion.html.twig

```diff
@@ -16,6 +16,14 @@
         <th>Key</th>
         <th>Value</th>
     </tr>
+    {% if versionNote %}
+        <tr class="system">
+            <td>Version Note</td>
+            <td>&nbsp;</td>
+            <td>{{ versionNote }}</td>
+        </tr>
+        <p><b>Version Note: </b> {versionNote}</p>
```

Comment:
having a row is enough...please kick it 
![image](https://user-images.githubusercontent.com/5137917/110756245-3b5be100-824a-11eb-81b4-334a7ed4a2ba.png) and please move it under "published". thanks!

Reply:
Fixed. The `<p>` was actually not supposed to be committed.